qiskit/transpiler/passes/optimization/template_matching/forward_match.py

```diff
@@ -404,13 +405,39 @@ def run_forward_match(self):
                 ):
                     continue
 
+                template = None
+
                 # Check if the qubit, clbit configuration are compatible for a match,
                 # also check if the operation are the same.
                 if (
                     self._is_same_q_conf(node_circuit, node_template)
                     and self._is_same_c_conf(node_circuit, node_template)
                     and self._is_same_op(node_circuit, node_template)
                 ):
+                ################################################
+                # Check if parameters match the template or not.
+                # parameter check까지 여기서 해버리자.
+                    temp_match = self.match.copy()
+                    temp_match.append([i,label])
+
+                    substitution = TemplateSubstitution(
+                        [temp_match],
+                        self.circuit_dag_dep,
+                        self.template_dag_dep,
+                        None,
+                    )
+                    
+                ##user_cost_dict = None 
```

Comment:
Looks like you left this here by accident.

Reply:
erased the "," and ##user_cost_dict.